content/en/network_performance_monitoring/installation.md

```diff
@@ -37,7 +37,9 @@ The following provisioning systems are supported:
 
 ## Setup
 
-To enable network performance monitoring, configure it in your [Agent's main configuration file][6] based on your system setup:
+To enable Network Performance Monitoring, configure it in your [Agent's main configuration file][6] based on your system setup. 
+
+Given this tool's focus and strength is in analyzing traffic _between_ network endpoints and mapping network dependencies, we recommend installing it on a meaningful subset of your infrastructure and a **_minimum of 2 hosts_** to maximize value. 
```

Comment:
```suggestion
Given this tool's focus and strength is in analyzing traffic _between_ network endpoints and mapping network dependencies, it is recommend to install it on a meaningful subset of your infrastructure and a **_minimum of 2 hosts_** to maximize value. 
```

Reply:
@sarina-dd good change! found one small typo: "it is recommendED to install it..."